### PR TITLE
Remove dotenv dependency, use process.loadEnvFile()

### LINKED ISSUE #31: remove dotenv dependency
Since Node 20.6, this functionality is built-in.

### PR DESCRIPTION
**Pull request by carlansley**
Fixes #31

### CODE REVIEW

**Review by ramaghanta — APPROVED** [review]
lftm

File: .github/workflows/check-published-scheduled.yml [hunk 1]
```diff
@@ -10,8 +10,7 @@ jobs:
     runs-on: ubuntu-latest
     name: Check Published
     steps:
-      - name: Checkout Code
-        uses: actions/checkout@v4
+      - uses: actions/checkout@v4
```
Comment (le-cong) [inline .github/workflows/check-published-scheduled.yml]: add back 'name'?

File: .github/workflows/ci.yml [hunk 2]
```diff
@@ -34,11 +31,11 @@ jobs:
         run: npm run ci:test
 
   branchBuild:
-    name: Branch Build
     runs-on: ubuntu-latest
+    name: Branch Build
```
Comment (le-cong) [inline .github/workflows/ci.yml]: move name before 'runs-on'?

File: .github/workflows/publish.yml [hunk 3]
```diff
@@ -12,13 +12,13 @@ jobs:
       id-token: write
     name: Publish to NPM
     steps:
-      - name: Checkout Code
-        uses: actions/checkout@v4
-      - name: Setup Node.js
-        uses: actions/setup-node@v4
+      - uses: actions/checkout@v4
```
Comment (le-cong) [inline .github/workflows/publish.yml]: add back name?

File: .github/workflows/publish-beta.yml [hunk 4]
```diff
@@ -23,8 +23,8 @@ jobs:
         uses: checkdigit/github-actions/check-label@main
         env:
           GITHUB_TOKEN: ${{ secrets.GITHUB_TOKEN }}
-      - name: Check Imports
-        uses: checkdigit/github-actions/check-imports@main
+      - name: Install latest npm
```
Comment (le-cong) [inline .github/workflows/publish-beta.yml:28]: should we keep Check Imports?

File: package.json [hunk 5]
```diff
@@ -18,21 +18,20 @@
   ],
   "type": "module",
   "engines": {
-    "node": ">=20.11"
+    "node": ">=20.12"
```
Comment (le-cong) [inline package.json:21]: the luke document has to be updated to specify node>=20.12 to make this work, right?

**Review by le-cong — CHANGES_REQUESTED** [review]
just a few minor CI config comments

File: .github/workflows/check-published-scheduled.yml (hunk 1 above)
Comment (carlansley) [reply .github/workflows/check-published-scheduled.yml]: i just re-copied everything from package standards, should be good now.

File: package.json (hunk 5 above)
Comment (carlansley) [reply package.json:21]: It probably should be updated at some point, but just updating to this package will be enough for `npm` to complain about the engine version, if you're running < 20.12.

**Review by le-cong — APPROVED** [review]
lgtm

**Pull request merged**